docs/querying/api.md

```diff
@@ -452,9 +452,10 @@ We also expose a gRPC API whose definition can be found [here](https://github.co
 
 ### Snapshot
 Snapshot creates a snapshot of all current data into `snapshots/<datetime>-<rand>` under the TSDB's data directory and returns the directory as response.
+It will optionally skip snapshotting data that is only present in memory and the write ahead log.
```

Comment:
This is hard to understand. Maybe: skip snapshotting data that is only present in the head block, and which has not yet been written out as a block.